plugins/source/aws/client/helpers.go

```diff
@@ -59,6 +59,7 @@ const (
 	SESService                  AWSService = "ses"
 	WAFRegional                 AWSService = "waf-regional"
 	WorkspacesService           AWSService = "workspaces"
+	IamService                  AWSService = "iam"
```

Comment:
I would keep this list sorted (move this line up after GuardDuty)

Reply:
fixed